sonic-slave-buster/Dockerfile.j2

```diff
@@ -491,6 +491,7 @@ RUN pip3 install git+https://github.com/aristanetworks/swi-tools.git@bead66bf261
 {% if CONFIGURED_ARCH != "amd64" -%}
 # Install node.js for azure pipeline
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
+RUN curl --silent https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add -
```

Comment:
Why this new line is required now? If it is a generic issue, we need to fix master first then cherry-pick.

Reply:
This is observed in the following azp job. I will send PR to master first.
https://dev.azure.com/mssonic/build/_build/results?buildId=24827&view=logs&j=594b531d-901a-5eaa-0051-20781037afc4&t=2d2c06c2-f02c-5153-72fa-81e97e4a6a65

Reply:
but this issue doesn't occur in our official build for sonic-slave-buster. Maybe it is transient? I will take care of it.